cwe_api/cvss_inferencer.py

```
@@ -0,0 +1,145 @@
+    with open(NUM_LABELS_PATH, 'rb') as handle:
+        num_labels = pickle.load(handle)
+
+    model = MultiOutputBert(num_labels).to(DEVICE)
+
+    model.load_state_dict(torch.load(MODEL_PATH, weights_only=False, map_location=torch.device('cpu')))
+    model.eval()
+
+    dataloader = generate_dataset(vuln)
+
+    with torch.no_grad():
+        for index, batch in enumerate(dataloader):
```

Ruff

Loop control variable `index` not used within loop body

Rename unused `index` to `_index`

(B007)